### DISCUSSION TITLE
Add custom CSS

### DISCUSSION THREAD

**Discussion by margomius** [JulianPrieber/littlelink-custom#375] [Questions]
Hi,
ist it possible to add custom CSS in the admin panel or via FTP?
If I wanted to change some CSS in the default theme, where would I have to change it?
It should work after updates as well.

Thanks in advance!

**Comment by JulianPrieber** [comment]
You can just do this by making a custom theme yourself. We got everything you need to make your own theme with a template of the default theme right [here](https://github.com/JulianPrieber/llc-themes/tree/main/contributing#readme)
